go-kit/metrics/provider/librato/librato_test.go

```diff
@@ -532,3 +538,47 @@ func TestWithResetCounters(t *testing.T) {
 		})
 	}
 }
+
+func TestWithResetCountersUniqueCounters(t *testing.T) {
+	//	for _, reset := range []bool{true, false} {
+	for _, reset := range []bool{true} {
```

Comment:
,false ?

Reply:
Ah. Yes. I was doing some debugging and forgot to back this one out.